server/src/printers/TestMakefilesPrinter.cpp

```diff
@@ -62,36 +74,52 @@ namespace printer {
         fs::path sharedMakefilePath = getMakefilePathForShared(generalMakefilePath);
         fs::path objMakefilePath = getMakefilePathForObject(generalMakefilePath);
 
+        generalMakefilePrinter.declareVariable("export project_dir_relative_to_makefile",
```

Comment:
move variable declare to here

Reply:
added RelativeMakefilePrinter entity for this